source/extensions/filters/http/cache/http_cache.cc

```diff
@@ -54,7 +54,7 @@ LookupRequest::LookupRequest(const Http::RequestHeaderMap& request_headers, Syst
 // Unless this API is still alpha, calls to stableHashKey() must always return
 // the same result, or a way must be provided to deal with a complete cache
 // flush.
-size_t stableHashKey(const Key& key) { return MessageUtil::hash(key); }
+size_t stableHashKey(const Key& key) { return DeterministicProtoHash::hash(key); }
```

Comment:
I guess we could reference a runtime flag check here to make it easy to flip modes if someting isn't quite right with the new hasher.

Reply:
ping on this; I think this depth of re-implementation needs an escape-hatch.

Reply:
Now added.